docs/overview.rst

```diff
@@ -366,6 +366,13 @@ TMT_PLUGINS
     Path to a directory with additional plugins. Multiple paths
     separated with the ``:`` character can be provided as well.
 
+TMT_FEELING_SAFE
+    Set this variable to ``True`` to enable potentially dangerous
```

Comment:
Hmmm, I think this might be incorrect, as an `envvar` key of a Click option, the allowed values would be `1` and `0`. `True` might work, because it's as a non-empty string it's true-ish, but `TMT_FEELING_SAFE=False` shouldn't.

Reply:
Seems that `click` is happy with any of these:

For enabling the flag:

* `1`
* `true`
* `t`
* `yes`
* `y`
* `on`

For disabling the flag:

* `0`
* `false`
* `f`
* `no`
* `n`
* `off`

We should probably choose one and use it consistently across the docs. Any preference?

Reply:
Good to know it supports more, I recall having trouble with at least some of them. Nevermind then.

The selection probably happened already, see the last paragraph in https://tmt.readthedocs.io/en/stable/overview.html#plugin-variables :)

Reply:
Ack, updated in 753c0cf8.